src/zope/app/apidoc/codemodule/browser/static_menu.pt

```diff
@@ -1,48 +1,39 @@
 <html metal:use-macro="context/@@apidoc_macros/static_menu"
-    i18n:domain="zope">
+      i18n:domain="zope">
 <body>
 
   <div metal:fill-slot="menu" class="small">
 
     <div>
-      <span i18n:translate="">Class Finder:</span>
-      <span><i i18n:translate="">(Enter partial Python path)</i></span>
-	</div>
-    <form action="#" method="post" name="searchform" >
-	  <br />
+      <span i18n:translate="">Class Finder:</span><br />
+      <em i18n:translate="">(Enter partial Python path)</em>
+    </div>
+    <form action="" method="post" name="searchform"
+          onsubmit="simplegetSearchResult(document.searchform.path.value);
```

Comment:
The changes to this file seem unrelated to the purpose of this PR. Can they be separated? 

(Actually I don't even understand why they would be necessary, this just seems to change one type of button to another. Then again, I haven't thought at this level in awhile.)